pkg/pool-manager/pool.go

```diff
@@ -99,19 +99,6 @@ func (p *PoolManager) getFreeMac() (net.HardwareAddr, error) {
 	return nil, fmt.Errorf("the range is full")
 }
 
-func (p *PoolManager) setAsLeader() {
```

Comment:
should be removed in the same commit you remove calls to this function

Reply:
the `setAsLeader` was removed In a different PR I forgot to remove the function there